### PR TITLE
revising env var usage

### PR DESCRIPTION
**Pull request by mattretzer**

### CODE REVIEW

**Comment by mattretzer** [comment]
Hi @nelliemckesson ,  just making two tweaks here to the travis setup for the addons branch..  my test for the environmental variable wasn't working as expected on the server, so I made it more specific; and I'm using an [environmental variable built-in to the Travis container](https://docs.travis-ci.com/user/environment-variables#Default-Environment-Variables) instead of our own from the .yml. 

FYI I've tested this on staging, and looked through logs as well as checking the generated epub

File: epubmaker_postprocessing.rb [hunk 1]
```diff
@@ -1,10 +1,11 @@
 require 'fileutils'
 require 'net/smtp'
 
-unless defined?(ENV['TRAVIS_TEST'])
+unless (ENV['TRAVIS']) == 'true'
```
Comment (nelliemckesson) [inline epubmaker_postprocessing.rb]: So where is the TRAVIS env variable defined? I don't see it in the yaml. Is it a system variable?

Comment (mattretzer) [inline epubmaker_postprocessing.rb]: It is:  https://docs.travis-ci.com/user/environment-variables#Default-Environment-Variables  We can put one back in the .yml if you'd rather?

Comment (nelliemckesson) [inline epubmaker_postprocessing.rb]: I think I'd prefer having it in the yml, just to eliminate one layer of admin, as long as it works as expected.

Comment (mattretzer) [inline epubmaker_postprocessing.rb]: I'll put it back & double-check!

**Comment by mattretzer** [comment]
ok, I put the environmental var back in the .yml; it passes travis and works on staging!  
Shall I go ahead and merge?

**Comment by nelliemckesson** [comment]
Yep, looks good! Make sure to run something through on production after, just to make sure it's all working there.

**Pull request merged**